src/Calculator/App.xaml.cs

```diff
@@ -179,234 +122,45 @@ private void OnAppLaunch(IActivatedEventArgs args, string argument)
 
             // Do not repeat app initialization when the Window already has content,
             // just ensure that the window is active
+            var rootFrame = Window.Current.Content as Frame;
             if (rootFrame == null)
             {
-                if (!Windows.Foundation.Metadata.ApiInformation.IsTypePresent("Windows.Phone.UI.Input.HardwareButtons")) // PC Family
-                {
-                    // Disable the system view activation policy during the first launch of the app
-                    // only for PC family devices and not for phone family devices
-                    try
-                    {
-                        ApplicationViewSwitcher.DisableSystemViewActivationPolicy();
-                    }
-                    catch (Exception)
-                    {
-                        // Log that DisableSystemViewActionPolicy didn't work
-                    }
-                }
-
-                // Create a Frame to act as the navigation context
-                rootFrame = App.CreateFrame();
-
-                // When the navigation stack isn't restored navigate to the first page,
-                // configuring the new page by passing required information as a navigation
-                // parameter
-                if (!rootFrame.Navigate(typeof(MainPage), argument))
+                rootFrame = new Frame
                 {
-                    // We couldn't navigate to the main page, kill the app so we have a good
-                    // stack to debug
-                    throw new SystemException();
-                }
-
-                SetMinWindowSizeAndThemeAndActivate(rootFrame, minWindowSize);
-                m_mainViewId = ApplicationView.GetForCurrentView().Id;
-                AddWindowToMap(WindowFrameService.CreateNewWindowFrameService(rootFrame, false, weak));
+                    FlowDirection = LocalizationService.GetInstance().GetFlowDirection()
+                };
```

Comment:
nit: use `App.CreateFrame()` directly.

Reply:
`CreateFrame` has been removed by this PR. The work of `CreateFrame` is not complicated and inline code may improve the fluency of reading.